README.md

```diff
@@ -95,6 +95,10 @@ annotation to your .proto file
     }
    ```
 
+   See [a_bit_of_everything.proto](examples/proto/examplepb/a_bit_of_everything.proto)
```

Comment:
Front and center in Usage seems like a good high-visibility place.

Reply:
Great, thanks!